test/routes/viewModels/taskListModelsCA.test.js

```diff
@@ -11,9 +11,9 @@ describe('TaskList models', () => {
     },
     label: 'You should now tell the offender using the relevant HDC form from NOMIS',
     title: 'Inform the offender',
-    visible: true,
+    visible: undefined,
```

Comment:
visibility on each individual task is no longer used. 

Will go through and remove them from the tasks and tests afterwards.